js/data/protokolle.json

```
@@ -10292,7 +10292,7 @@
     "sok": true,
     "datum": "1898-11-09",
     "dok": true,
-    "titel": "\\[\\md{f}{2}{y}{}{x}{}=\\md{f}{2}{x}{}{y}{}.\\]",
+    "titel": "$\frac{\partial^2 f}{\partial y\partial x}=\frac{\partial^2 f}{\partial x\partial y}$",
```

```suggestion
    "titel": "$\\frac{\\partial^2 f}{\\partial y\\partial x}=\\frac{\\partial^2 f}{\\partial x\\partial y}$",
```
Vielleicht geht es so?